.github/workflows/ci.yml

```diff
@@ -36,3 +36,19 @@ jobs:
 
       - name: Check formatting
         run: cargo fmt --all -- --check
+
+  clippy:
+    name: 'Clippy'
```

Comment:
Maybe call this "Check lints" or something like that? Clippy is the name of the tool used, but what this checks is lint errors.

Reply:
Good point, fixed